packages/pds/src/basic-routes.ts

```diff
@@ -25,7 +25,8 @@ export const createRouter = (ctx: AppContext): express.Router => {
       await sql`select 1`.execute(ctx.db.db)
     } catch (err) {
       req.log.error(err, 'failed health check')
-      return res.status(503).send({ version, error: 'Service Unavailable' })
+      res.status(500).send({ version, error: 'Service Unavailable' })
```

Comment:
Any special reason for the status code change?

Reply:
nope this was from a strange failure in one of the tests & i think i forgot to change it back